changelogs/unreleased/gh-5928-implement-several-sockets-with-different-properties.md

```diff
@@ -0,0 +1,20 @@
+## feature/core
+* Implement ability to pass several listening uries in different ways
+  with different properties. Currently the only valid option is `transport`
+  with `plain` value, with behaviour same as without this option. In the
+  future, new options will appear that will determine the behavior of iproto
+  threads. Fox example:
+  ```lua
+  box.cfg { listen = '10.10.0.1:3301' }
```

Comment:
Nit: not sure we should add so many examples to the changelog. I'd leave only the most complicated one